zenith_utils/src/lsn.rs

```diff
@@ -0,0 +1,175 @@
+#![warn(missing_docs)]
+
+use std::fmt;
+use std::ops::{Add, AddAssign};
+use std::path::Path;
+use std::str::FromStr;
+use std::sync::atomic::{AtomicU64, Ordering};
+
+// FIXME: should we implement conversions to/from PgLsn?
+// Doing so would mean zenith_utils gets a dependency on postgres-types.
+// That's not necessarily a bad thing, but if there is only one place
+// that needs it, it might be easier to do it there, e.g.
+//     let x = Lsn::from(u64::from(pglsn))
```

Comment:
I'm probably going to answer my own question here and say "no, let's not bother". There are very few places we need to interface with `postgres_types::PgLsn`, so I don't think this matters.

If I'm wrong, it's easy to fix later.

Reply:
Agreed